crates/ruff_python_stdlib/src/builtins.rs

```diff
@@ -200,7 +204,10 @@ pub fn python_builtins(minor: u8) -> Vec<&'static str> {
 /// Returns `true` if the given name is that of a Python builtin.
 ///
 /// Intended to be kept in sync with [`python_builtins`].
-pub fn is_python_builtin(name: &str, minor_version: u8) -> bool {
+pub fn is_python_builtin(name: &str, minor_version: u8, include_ipython_builtins: bool) -> bool {
```

Comment:
Nit: I would call the variable `is_stub`. The fact that it then must include the `ipython` builtins seems more like an implementation detail (the current variable tells it what it should do rather than what you want)

Reply:
Renamed it to `is_notebook`